stdlib/public/core/Arrays.swift.gyb

```diff
@@ -1812,6 +1792,32 @@ extension ${Self} {
 }
 %end
 
+// Utility method for collections that wish to implement CustomStringConvertible
+// and CustomDebugStringConvertible using a bracketed list of elements,
+// like an array.
+@_inlineable // FIXME(sil-serialize-all)
+@_versioned // FIXME(sil-serialize-all)
+internal func _makeArrayDescription<C : Collection>
```

Comment:
If we're going to use this for every collection, maybe we should call this `_makeCollectionDescription`

Reply:
Makes sense! Renamed.